### ISSUE TITLE
Loss value

### ISSUE THREAD

**Issue by Arcmoon-Hu** [garder14/simclr-tensorflow2#1]
I am trying run your code.
Can you please tell me the loss value when model end pretraining. 
parameters: 100 epochs,resnet 18,batch_size 512,tau 0.5
